### DISCUSSION TITLE
how can I export settings?

### DISCUSSION THREAD

**Discussion by chaserene** [wwmm/easyeffects#1653] [Q&A] [answered]
first of all, thank you for maintaining this software. it's freaking useful.

I'm using the PulseEffects 4.8.4 version, and want to export/save my settings to a file, with the intention of reusing it on another machine. if it's not possible to do for the whole application, I'd be fine with only the equalizer settings as well. how can I do this?

**Comment by wwmm** [comment, marked as answer]
Most of the settings that actually matter in a situation like this are saved to presets files. So you can save a preset and get it in the folder `~/.config/pulseeffects/` if PulseEffects was installed through an ordinary system package. If it was installed through Flatpak the preset files are saved somewhere else. I do not remember exactly where now.

**Reply by chaserene** [reply]
found it, thanks! a note for those who just want to copy-paste the path, the correct one is `~/.config/PulseEffects/` (it won't work with all lowercase).

**Comment by wwmm** [comment]
I think that in Flatpak the path is something like `~/.var/app/com.github.wwmm.pulseeffects/config/pulseeffects`.
